server/server_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"github.com/99designs/gqlgen/client"
+	"github.com/99designs/gqlgen/graphql/handler"
+
+	"go-gql-typescript-example/graph"
+	"go-gql-typescript-example/graph/generated"
+	"go-gql-typescript-example/graph/model"
+
+	"github.com/stretchr/testify/require"
+
+	"log"
+	"os"
+	"testing"
+)
+
+var user *model.User
+
+func TestMain(m *testing.M) {
+	graph.Database = openAndMigrateDatabase("file::memory:?cache=shared")
+
+	user = createUser("bob")
```

Comment:
this pattern will eventually bite someone, when you have many test files you'll want all the fixtures created in one spot.  In my bigger app I have a 'test' environment, and when in the 'test' environment an additional migration runs and appends all the fixture data.

Reply:
I will add that pattern in here at some point

Reply:
Would be handy to have database seeds for development as well. Be able to test `user(name: "bob"){id}` w/o running a mutation first.

Reply:
Exactly.  I like chipping away at this, moving patterns from my app into this example, so I'll do the environments pattern I've built sometime soon